GDJS/Runtime/inputmanager.js

```diff
@@ -69,22 +99,54 @@ gdjs.InputManager.prototype.getLastPressedKey = function() {
 
 /**
  * Return true if the key corresponding to keyCode is pressed.
- * @param {number} keyCode The key code to be tested.
+ * @param {number} modifiedKeyCode The key code (with location modifiers) to be tested.
  */
-gdjs.InputManager.prototype.isKeyPressed = function(keyCode) {
-  return (
-    this._pressedKeys.containsKey(keyCode) && this._pressedKeys.get(keyCode)
-  );
+gdjs.InputManager.prototype.isKeyPressed = function(modifiedKeyCode) {
+  if (modifiedKeyCode > 0) {
```

Comment:
For now I would suggest we avoid the "Any" key because I would like to avoid adding complexity on top of the already "complex" location-aware key code.

While it's not probable that in the future keyboards will change a lot, there will still be potential new APIs, or changes in the APIs, and I'd like to make our implementation as simple as possible and close to the notion of having a key code and a location.

Reply:
Sounds good!